lib/import/brca/core/brca_handler_mapping.rb

```diff
@@ -20,7 +20,8 @@ class BrcaHandlerMapping
           'RJ7' => Import::Brca::Providers::StGeorge::StGeorgeHandler,
           'RPY' => Import::Brca::Providers::RoyalMarsden::RoyalMarsdenHandler,
           'R1K' => Import::Brca::Providers::LondonKgc::LondonKgcHandler,
-          'RQ3' => Import::Brca::Providers::Birmingham::BirminghamHandlerNewformat
+          'RQ3' => Import::Brca::Providers::Birmingham::BirminghamHandlerNewformat,
+          'RP4' => Import::Brca::Providers::LondonGosh::LondonGoshHandler,
```

Comment:
@FrancescoNea Rubocop error introduced here:

Avoid comma after the last item of a hash. convention

Reply:
Thank you @ollietulloch , it should now be resolved

Reply:
Great, thanks - resolved in 2df6277